Cargo.toml

```diff
@@ -22,6 +22,12 @@ libc = "0.2.58"
 log = "0.4.7"
 pretty_env_logger = "0.3.0"
 once_cell = "0.2.4"
+nodrop = "0.1.14"
+lazy_static = "1.4.0"
```

Comment:
I believe you can use `once_cell` instead, which is already a dependency

Reply:
fixed